utilities.py

```diff
@@ -0,0 +1,20 @@
+import uuid
+from typing import List
+
+from history.exceptions import UnknownHistoryEntryUUID
+from history.models import HistoryEntry
+
+__all__ = [
+    "get_history_entries",
+]
+
+
+def get_history_entries(object_uuid: uuid) -> List[HistoryEntry]:
```

Comment:
This is a utility function, I would return tuples of datetime and string (to make it model-agnostic)